src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/LocationMeta.cs

```diff
@@ -1,10 +1,52 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Converters;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
 using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
 
 namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
 
-public class LocationMeta
+public class LocationMeta : ICreatedUpdatedTimestamps<DateTimeOffset, DateTimeOffset?>
 {
-    public required GeographicLevel Level { get; set; }
+    public int Id { get; set; }
```

Comment:
I think we said we'd stick to using `Guids`?

Reply:
**Discussed:**

Only using `Guids` for publically available IDs. Using `int` for strictly internally available IDs as it is more performant.